common/app/implicits/FaciaContentImplicits.scala

```diff
@@ -92,11 +92,11 @@ object FaciaContentImplicits {
     def blogs: Seq[Tag] = FaciaContentUtils.blogs(fc)
     def tones: Seq[Tag] = FaciaContentUtils.tones(fc)
     def types: Seq[Tag] = FaciaContentUtils.types(fc)
-    def isVideo = FaciaContentUtils.isVideo(fc)
-    def isGallery = FaciaContentUtils.isGallery(fc)
+    def isVideo: Boolean = FaciaContentUtils.isVideo(fc)
```

Comment:
Is it worth adding the type on the rest of these boolean `is ___` implicits for consistency's sake?